apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -1510,7 +1498,7 @@ fn request_ftv1(mut req: SubgraphRequest) -> SubgraphRequest {
         .private_entries
         .lock()
         .contains_key::<EnableSubgraphFtv1>()
-        && !Span::current().is_disabled()
+        && Span::current().context().span().span_context().is_sampled()
```

Comment:
This looks like it was not a chance in the changeset of the original PR.

https://github.com/apollographql/router/pull/2894/files#diff-37adf9e170c9b384f17336e5b5e5bf9cd94fd1d618b8969996a5ad56b635ace6L624-R644